plugins/elantp/fu-elantp-hid-device.c

```diff
@@ -216,6 +241,8 @@ fu_elantp_hid_device_setup(FuDevice *device, GError **error)
 	}
 	self->module_id = fu_memread_uint16(buf, G_LITTLE_ENDIAN);
 
+	fu_device_set_vendor(device, "ELAN Microelectronics Corp.");
```

Comment:
ohh? What value is there automatically, i.e. what are you replacing?

Reply:
Ahh, so it's not found the vendor and is falling back to the Vendor ID. Maybe just drop the "Corp." and then this change is okay.